src/ServiceProviderLaravel5.php

```diff
@@ -26,7 +26,7 @@ public function register()
     {
         $this->mergeConfigFrom(__DIR__.'/config/config.php', 'slack');
 
-        $this->app['maknz.slack'] = $this->app->share(function ($app) {
+        $this->app['maknz.slack'] = $this->app->singleton('maknz.slack', function () {
```

Comment:
The `singleton` method doesn't return anything, so it shouldn't be assigned to that Container key.